exercises/collatz-conjecture/test_collatz-conjecture.R

```diff
@@ -0,0 +1,28 @@
+source("./collatz-conjecture.R")
+library(testthat)
+
+test_that("Input of one results in zero steps", {
```

Comment:
I had to read this several times, because `of one results` seemed strange. I guess it's grammatically correct, but how about using just `1` here, not the word. Same for the `of zero results` below.